exe/kubernetes-deploy

```diff
@@ -48,6 +47,7 @@ end
 namespace = ARGV[0]
 context = ARGV[1]
 logger = KubernetesDeploy::FormattedLogger.build(namespace, context, verbose_prefix: verbose_log_prefix)
+KubernetesDeploy::StatsD.build(logger)
```

Comment:
Usually logs and metrics go hand in hand so this seemed like a natural place. Dont feel strongly about it so i'll move to `kubernetes-deploy.rb`